lib/initial-setup/WP_Auth0_InitialSetup_Connections.php

```diff
@@ -191,7 +191,7 @@ public function callback() {
       try {
         $input = $operations->social_validation( $this->a0_options->get( 'auth0_app_token' ), $old_input, $input, $provider_name, $provider['options'] );
       } catch ( Exception $e ) {
-        die( $e->getMessage() );
+        exit( $e->getMessage() );
```

Comment:
`die` `wp_die` and `exit` are analogous ?

Reply:
yes

Reply:
`die` and `exit` are, `wp_die` ends the process with HTML output and a 500 error response.